File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Phys/PHY2001U04GrdOut1001Info.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Phys
{
    [Serializable]
    public class PHY2001U04GrdOut1001Info
    {
        private String _bunho;
        private String _naewonDate;
        private String _jubsuGubun;
        private String _next;

        public String Bunho
        {
            get { return this._bunho; }
            set { this._bunho = value; }
        }

        public String NaewonDate
        {
            get { return this._naewonDate; }
            set { this._naewonDate = value; }
        }

        public String JubsuGubun
        {
            get { return this._jubsuGubun; }
            set { this._jubsuGubun = value; }
        }

        public String Next
        {
            get { return this._next; }
            set { this._next = value; }
        }

        public PHY2001U04GrdOut1001Info() { }

        public PHY2001U04GrdOut1001Info(String bunho, String naewonDate, String jubsuGubun, String next)
        {
            this._bunho = bunho;
            this._naewonDate = naewonDate;
            this._jubsuGubun = jubsuGubun;
            this._next = next;
        }

    }
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/phys/PHY2001U04GrdOut1001Info.java
package nta.med.data.model.ihis.phys;

public class PHY2001U04GrdOut1001Info {
	private String bunho                  ;
	private String naewonDate            ;
	private String jubsuGubun            ;
	private String next                   ;
	public PHY2001U04GrdOut1001Info(String bunho, String naewonDate,
			String jubsuGubun, String next) {
		super();
		this.bunho = bunho;
		this.naewonDate = naewonDate;
		this.jubsuGubun = jubsuGubun;
		this.next = next;
	}
	public String getBunho() {
		return bunho;
	}
	public void setBunho(String bunho) {
		this.bunho = bunho;
	}
	public String getNaewonDate() {
		return naewonDate;
	}
	public void setNaewonDate(String naewonDate) {
		this.naewonDate = naewonDate;
	}
	public String getJubsuGubun() {
		return jubsuGubun;
	}
	public void setJubsuGubun(String jubsuGubun) {
		this.jubsuGubun = jubsuGubun;
	}
	public String getNext() {
		return next;
	}
	public void setNext(String next) {
		this.next = next;
	}
	

}
